### DISCUSSION TITLE
슬라이드의 "*0"가 에러 상태를 의미하나요? ("*0" in slide 5)

### DISCUSSION THREAD

**Discussion by 33577** [prosyslab-classroom/cs524-program-analysis#446] [Q&A] [answered]
### Your Name

전종욱

### Question

안녕하세요 :)

5단원 슬라이드의 6-11 페이지에 나오는 그림에 대해 질문이 있습니다. 
빨간색 원은 오류 상태를 의미하는 것 같습니다. MAX++는 정수 오버플로우, arr[size]는 버퍼 오버플로우일 것 같습니다. 
그런데 "*0"은 어떤 오류 상태인지, 혹시 "/0" (또는 *null)의 오타인지 궁금합니다. 

감사합니다. 

<img width="1385" alt="image" src="https://github.com/prosyslab-classroom/cs524-program-analysis/assets/34697855/8ab8ceb5-7998-4b9f-8d43-131874460b6e">

**Comment by KihongHeo** [comment, marked as answer]
C 포인터에서 0 = null 이라는 것을 이용한 비유입니다. 

Null dereference in C.

**Reply by 33577** [reply]
아 그렇군요 감사합니다!!

Thanks for your help :)
